boot/freeldr/freeldr/arch/i386/mb.S

```diff
@@ -30,9 +29,8 @@ PUBLIC _kernel_pagetable
 PUBLIC _apic_pagetable
 PUBLIC _PageDirectoryEnd
 
-#ifndef _USE_ML
 .bss
-#endif
+
```

Comment:
That thing can compile with MASM; but it appears no definitions from there are actually used in FreeLdr at all.....
Leftover?

Reply:
Does GCC build complain, when it's removed?